traffic_monitor/tests/integration/README.md

```diff
@@ -0,0 +1,12 @@
+# Traffic Monitor Integration Test Framework
+
+## Running
+
+From the `trafficcontrol/traffic_monitor` directory:
+
+```
+(cd tools/testto && go build)
+(cd tools/testcaches && go build)
+(cd tests/integration && go test -c -o traffic_monitor_integration_test)
```

Comment:
When I run the docker-compose command to start the test,

```shell
docker-compose -p tmi --project-directory . -f tests/integration/docker-compose.yml run tmintegrationtest
```

I indefinitely get

```
Waiting for Traffic Ops to return a 200 OK
```

Looking at the `testto` container, it says

```logs
[user@computer traffic_monitor]$ 3 tmi --project-directory . -f tests/integration/docker-compose.yml logs -f testto
Attaching to tmi_testto_1
testto_1             | testto: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by testto)
tmi_testto_1 exited with code 1
```

So the instructions should have you build with `CGO_ENABLED=0` (and `GOOS=linux`).